ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -346,9 +346,12 @@ def process_experiences(
                 else:
                     bootstrapping_info = info
                     idx = l
-                value_next = self.policy.get_value_estimates(bootstrapping_info, idx)
-                if info.local_done[l] and not info.max_reached[l]:
-                    value_next["extrinsic"] = 0.0
+                value_next = self.policy.get_value_estimates(
+                    bootstrapping_info,
+                    idx,
+                    info.local_done[l] and not info.max_reached[l],
```

Comment:
Is it possible to determine this just from `bootstrapping_info` and `idx` within `get_value_estimates()`? I couldn't quite convince myself when I was looking at it.

Reply:
Hmm, I'm not sure - it seems like `bootstrapping_info` becomes the previous `info` if the `not` condition is met. Seems like we'll run into an issue if both of those conditions are met, since `bootstrapping_info` will be sth different and we can't figure out if `info.local_done` is True.

Reply:
OK, that's more or less what I thought. Sounds good as it is!